test/test_utils.py

```diff
@@ -6,15 +6,72 @@
 import sys
 from threading import Thread
 import time
+from typing import List
 from unittest import mock
 
 from flaky import flaky
 from docstring_to_markdown import UnknownFormatError
+from pylsp.lsp import NotebookCellKind
 
 from pylsp import _utils
```

Comment:
```suggestion

from pylsp import _utils
from pylsp.lsp import NotebookCellKind
```

Move to the local imports section